src/compiler/checker.ts

```diff
@@ -46321,7 +46333,7 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
                 return grammarErrorOnNode(node, message);
             }
 
-            if (isImportDeclaration(declaration) ? declaration.importClause?.isTypeOnly : declaration.isTypeOnly) {
+            if (isImportDeclaration(declaration) || isJSDocImportTag(declaration) ? declaration.importClause?.isTypeOnly : declaration.isTypeOnly) {

```

Comment:
Since jsdoc imports are *always* type-only, shouldn't this error always be issued, regardless of weather the input clause had a `type` modifier or not?

Reply:
Yes, you are right. Thanks

Reply:
@weswigham could you review the latest changes? Thanks